CMakeLists.txt

```diff
@@ -53,8 +54,10 @@ if(SILKIT_BUILD_LINUX_PACKAGE)
     message(NOTICE "  * SILKIT_BUILD_REPRODUCIBLE   OFF")
     set(SILKIT_BUILD_REPRODUCIBLE OFF CACHE BOOL "Force SILKIT_BUILD_REPRODUCIBLE OFF due to SILKIT_BUILD_LINUX_PACKAGE" FORCE)
 
-    message(NOTICE "  * SILKIT_USE_SYSTEM_LIBRARIES ON")
-    set(SILKIT_USE_SYSTEM_LIBRARIES ON CACHE BOOL "Force SILKIT_USE_SYSTEM_LIBRARIES ON due to SILKIT_BUILD_LINUX_PACKAGE" FORCE)
+    if(NOT SILKIT_VENDORED_PACKAGE)
+        message(NOTICE "  * SILKIT_USE_SYSTEM_LIBRARIES ON")
+        set(SILKIT_USE_SYSTEM_LIBRARIES ON CACHE BOOL "Force SILKIT_USE_SYSTEM_LIBRARIES ON due to SILKIT_BUILD_LINUX_PACKAGE" FORCE)
+    endif()
```

Comment:
Instead of adding another `option`, we could also just not `set(SILKIT_USE_SYSTEM_LIBRARIES ...)` at all. The `SILKIT_USE_SYSTEM_LIBRARIES` option is exactly the negation of the new `SILKIT_VENDORED_PACKAGE` when building a package.

```suggestion
```

Reply:
Less options is definately more good. Let me quickly reconstruct the logic and then I will fix it :)